### PR TITLE
add genesis export for lockup module

### PR DESCRIPTION
**Pull request by antstalepresh**

### CODE REVIEW

File: x/lockup/genesis.go [hunk 1]
```diff
@@ -9,9 +9,22 @@ import (
 // InitGenesis initializes the capability module's state from a provided genesis
 // state.
 func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
+	k.SetLastLockID(ctx, genState.LastLockId)
+	for _, lock := range genState.Locks {
+		if err := k.ResetLock(ctx, lock); err != nil {
```
Comment (ValarDragon) [inline x/lockup/genesis.go:14]: Lets add a comment for what is ResetLock here

File: app/app.go [hunk 2]
```diff
@@ -297,6 +297,21 @@ func NewOsmosisApp(
 	)
 	app.UpgradeKeeper = upgradekeeper.NewKeeper(skipUpgradeHeights, keys[upgradetypes.StoreKey], appCodec, homePath)
 
+	// this configures a no-op upgrade handler for the "upgrade-lockup-module-store-management" upgrade
+	app.UpgradeKeeper.SetUpgradeHandler("upgrade-lockup-module-store-management", func(ctx sdk.Context, plan upgradetypes.Plan) {
+		locks, err := app.LockupKeeper.GetPeriodLocks(ctx)
```
Comment (ValarDragon) [inline app/app.go:302]: This approach looks right to me, just want to double check via a test that there isn't some edge case we're missing

Comment (antstalepresh) [reply app/app.go]: Added test

File: x/lockup/genesis.go (hunk 1 above)
Comment (antstalepresh) [reply x/lockup/genesis.go:15]: Added comment

**Comment by ValarDragon** [comment]
Tested yesterday, and LGTM.

**Pull request merged**